requirements-flake8.txt

```diff
@@ -0,0 +1,2 @@
+# Requirements for running flake8
+flake8==6.0.0
```

Comment:
flake8 has dependency conflicts with other packages, so I moved it into its own environment.